variables-workers.tf

```diff
@@ -0,0 +1,232 @@
+}
+
+variable "output_worker_detail" {
+  default     = false
+  description = "Whether to include detailed output for worker pool configuration."
+  type        = bool
+}
+
+#
+# Workers: network
+#
+
+variable "worker_type" {
+  default     = "private"
+  description = "Whether to provision public or private workers."
+  type        = string
+  validation {
+    condition     = contains(["public", "private"], var.worker_type)
+    error_message = "Accepted values are public or private."
+  }
+}
+
+variable "worker_nsg_ids" {
+  default     = []
+  description = "An additional list of network security group (NSG) IDs for node security."
+  type        = list(string)
+}
+
+variable "pod_nsg_ids" {
+  default     = []
+  description = "An additional list of network security group (NSG) IDs for pod security."
+  type        = list(string)
+}
+
+variable "kubeproxy_mode" {
+  default     = "iptables"
+  description = "The mode in which to run kube-proxy."
+  type        = string
+
+  validation {
+    condition     = contains(["iptables", "ipvs"], var.kubeproxy_mode)
+    error_message = "Accepted values are iptables or ipvs."
+  }
+}
+
+#
+# Workers: instance
+#
+
+variable "worker_block_volume_type" {
+  default     = "paravirtualized"
+  description = "The default block volume attachment type for Instance Configurations."
+  type        = string
+  validation {
+    condition     = contains(["iscsi", "paravirtualized"], var.worker_block_volume_type)
+    error_message = "Accepted values are 'iscsi' or 'paravirtualized'."
+  }
+}
+
+variable "worker_node_labels" {
+  default     = {}
+  description = "Default worker node labels. Merged with labels defined on each pool."
+  type        = map(string)
+}
+
+variable "worker_image_id" {
+  default     = null
+  description = "Default image for worker pools when unspecified"
+  type        = string
+}
+
+variable "worker_image_type" {
+  default     = "oke"
+  description = "Whether to use a platform, OKE, or custom image for worker nodes by default. When custom is set, the worker_image_id must be specified."
+  type        = string
+  validation {
+    condition     = contains(["custom", "oke", "platform"], var.worker_image_type)
+    error_message = "Accepted values are custom, oke, platform"
+  }
+}
+
+variable "worker_image_os" {
+  default     = "Oracle Linux"
+  description = "Default worker image operating system name when worker_image_type = 'oke' or 'platform'."
+  type        = string
+}
+
+variable "worker_image_os_version" {
+  default     = "8"
+  description = "Default worker image operating system version when worker_image_type = 'oke' or 'platform'."
+  type        = string
+}
+
+variable "worker_shape" {
+  default = {
+    shape            = "VM.Standard.E4.Flex",
+    ocpus            = 2,
+    memory           = 16,
+    boot_volume_size = 50
+  }
+  description = "Default shape of the created worker instance."
+  type        = map(any)
+}
+
+variable "worker_cloudinit" {
```

Comment:
Is this the default cloud init?

Reply:
Yes, clarified description for behavior.